index.js

```diff
@@ -113,7 +148,14 @@ function findAllDependenciesHelp(file, knownDependencies, baseDir, knownFiles) {
           }
 
           // e.g. ~/code/elm-css/src/Css/Declarations.elm
-          var result = path.join(baseDir, dependencyLogicalName + extension);
+          var result = null;
+          _.find(sourceDirectories, function(sourceDir) {
+            var absPath = path.join(sourceDir, dependencyLogicalName + extension);
+            if (fs.existsSync(absPath)) {
+              result = absPath;
```

Comment:
The mutation avoids having to either `map` all entries to absolute paths, or having to map the found result a second time after returning it. I'm happy to go with either one if you consider this an unwanted optimisation.